### ISSUE TITLE
WCAG Accessibility Testing

### ISSUE THREAD

**Issue by valendesigns** [valendesigns/archetype#81]
Use [tenon.io](http://tenon.io) to test the theme before releasing it. This should be one of the last things that is done.
- [ ] Create a list of issues
- [ ] Create tickets for individual or related groups of issues.
- [ ] Close all issues!
